### PR TITLE
More RAV4_TSS2 FW

### PR DESCRIPTION
**Pull request by arne182**
dongle_id: 00dc05336f1756d9

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -710,6 +711,7 @@ class CAR:
       b'\x028646F4203300\x00\x00\x00\x008646G26011A0\x00\x00\x00\x00',
       b'\x028646F4203400\x00\x00\x00\x008646G2601200\x00\x00\x00\x00',
       b'\x028646F4203500\x00\x00\x00\x008646G2601200\x00\x00\x00\x00',
+      b'\x028646F4203500\x00\x00\x00\x00!!!!!!!!!!!!!!!!',
```
Comment (vanillagorillaa) [inline selfdrive/car/toyota/values.py:714]: I show `\x028646F4203500\x00\x00\x00\x008646G2601200\x00\x00\x00\x00` in the logs for this fw for this vehicle, which matches the FW already in the list

Comment (vanillagorillaa) [reply selfdrive/car/toyota/values.py:714]: Can you share the specific drive where this FW was taken from?

Comment (arne182) [reply selfdrive/car/toyota/values.py]: FROM Event 234d57e166f3444aba81026062494a66
Apr 3, 2020 12:02:50 PM UTCJSON (2.4 KB)
TAGS
branch:origin/release4
dirty:False
dongle_id: 00dc05336f1756d9
0.7.4-arne182
remote:https://github.com/arne182/openpilot
car doesn't match any fw: [<car.capnp:CarParams.CarFw builder (ecu = esp, fwVersion = "\x01F152642551\x00\x00\x00\x00\x00\x00", address = 1968, subAddress = 0)>, <car.capnp:CarParams.CarFw builder (ecu = engine, fwVersion = "\x018966342U4000\x00\x00\x00\x00", address = 1792, subAddress = 0)>, <car.capnp:CarParams.CarFw builder (ecu = eps, fwVersion = "8965B42170\x00\x00\x00\x00\x00\x00", address = 1953, subAddress = 0)>, <car.capnp:CarParams.CarFw builder (ecu = fwdRadar, fwVersion = "\x018821F3301200\x00\x00\x00\x00", address = 1872, subAddress = 15)>, <car.capnp:CarParams.CarFw builder (ecu = fwdCamera, fwVersion = "\x028646F4203500\x00\x00\x00\x00!!!!!!!!!!!!!!!!", address = 1872, subAddress = 109)>]

**Comment by arne182** [comment]
So only x018966342U4000\x00\x00\x00\x00 for Ecu.engine

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (pd0wm) [reply selfdrive/car/toyota/values.py]: Looks more like a sentry issue not rendering the characters correctly.

**Comment by pd0wm** [comment]
I can only find the ! in a single drive `00dc05336f1756d9|2020-04-03--08-02-45`. Weird.

**Comment by arne182** [comment]
You tell me what is up with that. So it is not my sentry displaying an error

**Pull request merged**